src/mounts.rs

```diff
@@ -155,39 +159,42 @@ pub fn finish_rootfs(spec: &Spec) -> Result<()> {
 lazy_static! {
     static ref OPTIONS: HashMap<&'static str, (bool, MsFlags)> = {
         let mut m = HashMap::new();
```

Comment:
this reformat is somewhat unfortunate, since i find the alighned version to be more readable. I suppose rustfmt did this though?

Reply:
I just ran `cargo fmt` to make it consistent after doing the rebase, and I'd recommend to stick to that just to not run into manual formatting messes though it might not be perfect in certain cases (like this) - but this might be just my personal opinion.

Reply:
Added a `rustfmt_skip` section and reformatted it.